petitions/backends.py

```diff
@@ -0,0 +1,22 @@
+from django_cas_ng.backends import CASBackend
+from django.conf import settings
+import requests
+import os
+
+
+class StudentCASBackend(CASBackend):
+    def user_can_authenticate(self, user):
```

Comment:
A debug mode would be helpful here so that we can test with different users or bypass cas entirely

Reply:
I searched around, and there doesn't seem like an easy way to do this. For now, I added that if debug mode is true, then the user is set to a superuser, which at least avoids accessing the database to set permissions. 

I think what you suggested is a good idea, but I think we can create an issue for this, as opposed to including it in this PR.